config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -239,6 +239,8 @@ presubmits:
         - --env=KUBE_NODE_OS_DISTRIBUTION=ubuntu
         - --env=KUBE_GCE_NODE_IMAGE=ubuntu-2004-focal-v20200423
         - --env=KUBE_GCE_NODE_PROJECT=ubuntu-os-cloud
+        - --env=KUBE_CONTAINER_RUNTIME=docker
+        - --env=ENABLE_POD_SECURITY_POLICY=true
```

Comment:
wait… why are we turning this on in presubmit?

Reply:
this is the optional presubmit CI job @liggitt (confusing i know :) )

the blocking one is `pull-kubernetes-e2e-gce-ubuntu-containerd` (please see https://testgrid.k8s.io/presubmits-kubernetes-blocking)

Reply:
But PodSecurityPolicy is optional and deprecated. We shouldn't be enabling it in new jobs. No e2e or addon should fail if it's not enabled.

Reply:
good point @liggitt 

@aojea can you please file a PR to roll that back?

Reply:
> good point @liggitt
> 
> @aojea can you please file a PR to roll that back?

I was copying pasting from the other job, let me fix it

Reply:
Ok, this fixed the broken job.
The important thing here is that the periodic ( that is release informing and has several issues open) and the presubmit are equal. Should I remove pod security policy in both?

Reply:
or should I remove it everywhere in master?
```
grep -r ENABLE_POD_SECURITY_POLICY config/
config/jobs/kubernetes/generated/generated.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/generated/generated.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml:          - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-release/release-branch-jobs/1.20.yaml:        - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-release/release-branch-jobs/1.20.yaml:        - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-release/release-branch-jobs/1.21.yaml:        - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-release/release-branch-jobs/1.21.yaml:        - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml:        - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml:        - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:  - name: ENABLE_POD_SECURITY_POLICY
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:        - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:            - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:            - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:          - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:          - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-node/containerd.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-node/containerd.yaml:          - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-node/containerd.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-node/containerd.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-node/sig-node-config.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
config/jobs/kubernetes/sig-testing/coverage.yaml:          --env=ENABLE_POD_SECURITY_POLICY=true \
config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml:      - --env=ENABLE_POD_SECURITY_POLICY=true
```